specifyweb/frontend/js_src/lib/components/FormParse/fields.ts

```diff
@@ -203,6 +204,7 @@ const processFieldType: {
       step: f.parseFloat(getProperty('step')),
       minLength: f.parseInt(getProperty('minLength')),
       maxLength: f.parseInt(getProperty('maxLength')),
+      isBold: getProperty('isBold')?.toLowerCase() === 'true',
```

Comment:
also, if you look at sp6's behavior, if I remember correctly, the default value for whether form is bold is not always false - it might depend on whether field is required, or field type or etc, don't remember
not saying we have to make it work exactly the same way in sp7, but saying that we should at least discuss that